src/phmdoctest/direct.py

```diff
@@ -9,8 +9,9 @@
 class Marker(Enum):
     """HTML comment before a fenced code block."""
     SKIP = '<!--phmdoctest-skip-->'
+    SKIPIF = '<!--phmdoctest-skipif<3.'   # No trailing space, no "-->".
     MARK_SKIP = '<!--phmdoctest-mark.skip-->'
-    SKIPIF = '<!--phmdoctest-mark.skipif<3.'   # No trailing space, no "-->".
+    MARK_SKIPIF = '<!--phmdoctest-mark.skipif<3.'   # No trailing space, no "-->".
```

Comment:
I assume this was typo in naming

Reply:
There should only be one enumeration intended to generate @pytest.mark.skipif in the outfile.
I see two here: SKIPIF and MARK_SKIPIF.